uniffi_bindgen/src/bindings/python/templates/EnumTemplate.py

```diff
@@ -136,16 +136,16 @@ def check_lower(value):
     def write(value, buf):
         {%- for variant in e.variants() %}
         {%- if e.is_flat() %}
-        if value == {{ type_name }}.{{ variant.name()|enum_variant_py }}:
+        if value == {{ type_name }}.{{ variant.name()|enum_variant_py(config) }}:
             buf.write_i32({{ loop.index }})
         {%- else %}
-        if value.is_{{ variant.name()|var_name }}():
+        if value.is_{{ variant.name()|var_name}}():
```

Comment:
Nano-nit: keep the space before the `}}` here and in some other locations.  That's generally how we format things and also will keep the commit size down.

Reply:
Ups, yeah, oversight through Search+Replace.